parser/src/java/org/apache/hadoop/hive/ql/parse/AlterClauseParser.g

```diff
@@ -686,6 +687,13 @@ alterStatementSuffixCreateOrReplaceTag
      -> ^(TOK_ALTERTABLE_CREATE_TAG $tagName KW_REPLACE snapshotIdOfRef? refRetain?)
      ;
 
+alterStatementSuffixSetWriteZOrder
```

Comment:
can't we allow command without zorder? why not reuse `tableWriteLocallyOrderedBy` ?

Reply:
I am going to have a PR to support alter table for WRITE ORDER as well.
The logic for that is different than zorder as that will require values to be added in iceberg spec and not in HMS props.
So, Based on the need for ordered(lexical) the common code names will be changed.